python/ray/tune/automl/search_space.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, name):
         """Initialize ParameterSpace.
 
         Arguments:
-            name (str): Name of the parameter. Name can be dot separated,
+            name: Name of the parameter. Name can be dot separated,
```

Comment:
In the cases where we remove the type annotation in the docstring, we should add it to the function definition instead, e.g. `def __init__(self, name: str):`